resources/assets/js/components/Gif.vue

```diff
@@ -11,15 +11,22 @@
 </style>
 
 <template>
-  <div class="gif-container" @click="$emit('click')">
-    <span v-if="starEnabled" class="star">{{starred}}</span>
+  <div class="gif-container" @click="toggleStar">
+    <span v-if="isConnected" class="toggleStar">{{starred}}</span>
     <img :src="src" />
   </div>
 </template>
 
 <script>
+import store from '../store.js'
```

Comment:
Une fois l'injection du store effectuée dans le composant de base, il est accessible dans tous ses enfants par `this.$store`